pkg/heroku/heroku.go

```diff
@@ -62,23 +62,43 @@ type Client struct {
 }
 
 func (c *Client) Get(v interface{}, path string) error {
-	return c.APIReq(v, "GET", path, nil)
+	return c.APIReq(v, "GET", path, nil, nil)
 }
 
 func (c *Client) Patch(v interface{}, path string, body interface{}) error {
-	return c.APIReq(v, "PATCH", path, body)
+	return c.APIReq(v, "PATCH", path, body, nil)
 }
 
 func (c *Client) Post(v interface{}, path string, body interface{}) error {
-	return c.APIReq(v, "POST", path, body)
+	return c.APIReq(v, "POST", path, body, nil)
 }
 
 func (c *Client) Put(v interface{}, path string, body interface{}) error {
-	return c.APIReq(v, "PUT", path, body)
+	return c.APIReq(v, "PUT", path, body, nil)
 }
 
 func (c *Client) Delete(path string) error {
-	return c.APIReq(nil, "DELETE", path, nil)
+	return c.APIReq(nil, "DELETE", path, nil, nil)
+}
+
+func (c *Client) GetWithHeaders(v interface{}, path string, headers http.Header) error {
```

Comment:
i went with the `WithHeaders` suffix to avoid having to go through and add a `nil` parameter to every existing invocation of these functions.